borg/testsuite/key.py

```diff
@@ -4,11 +4,14 @@
 import tempfile
 from binascii import hexlify, unhexlify
 
+from borg.logger import setup_logging
 from ..crypto import bytes_to_long, num_aes_blocks
 from ..key import PlaintextKey, PassphraseKey, KeyfileKey
 from ..helpers import Location
 from . import BaseTestCase
 
+setup_logging()
```

Comment:
Got a few failures due to un-initialized loggers when running this one alone without running the entire suite. When running the test suite the archiver test cases will do this.

Maybe put it in __init__ of the test suite instead, to ensure that the loggers are always there for all tests?